apps/deploy-web/src/utils/domUtils.ts

```diff
@@ -3,3 +3,17 @@ export function hasSomeParentTheClass(element: HTMLElement, classname: string):
   if (element.className?.split && element.className.split(" ").indexOf(classname) >= 0) return true;
   return !!element.parentNode && hasSomeParentTheClass(element.parentNode as HTMLElement, classname);
 }
+
+/**
+ * Adds a script tag to the document head
+ * @param src The source URL of the script
+ * @param async Whether the script should be loaded asynchronously
+ * @returns The created script element
+ */
+export function addScriptToHead(src: string, async: boolean = true): HTMLScriptElement {
```

Comment:
suggestion(non-blocking): better to pass options object because we could reuse this in different contexts. For example there may be cors restrictions or might need some data attribute on script, etc